### ISSUE TITLE
Record sound. 

### ISSUE THREAD

**Issue by NripPrajapat** [Angel-Studio/SoundTap#10]
I know the app doesn't suggest that idea.
But it would be cool to have a feature that mimics the iPhone 'action button'.
Three more actions for customize actions. 
Record, pause and save.

Thanks, give it a thought At least.

**Comment by JulesPvx** [comment]
Yeah, why not, that's a good idea. I'll check if it's actually possible :)

**Comment by JulesPvx** [comment]
Not so simple, need to implement my own recorder since there's no way to trigger the system one. I will still add it in the next update I think :)
